src/styles/components/home-page.styl

```diff
@@ -0,0 +1,189 @@
+@import "font-styles.styl"
```

Comment:
I was going to ask "why do we need to have `@import "font-styles.styl` here?" but then I realised it's going to be something related to a ridiculous quirk about code precedence, blurgh. OK, I'll write a full comment about this.